Makefile

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 
 REGISTRY_NAME=quay.io/k8scsi
-IMAGE_NAME=hostpathplugin
```

Comment:
nfs and iscsi plugins have also moved to their own repo.  @pohly can you help make sure the image building is setup properly in the new repos, and then we can just remove all of the drivers in this PR?

Reply:
Okay. See https://github.com/kubernetes-csi/csi-driver-nfs/pull/5 and https://github.com/kubernetes-csi/csi-driver-iscsi/pull/6.

Reply:
@msau42 @pohly what about moving one by one or in a small group , instead of everything in a PR ?